docs/spec/v1beta1/imagerepositories.md

```diff
@@ -190,7 +221,7 @@ type ImageRepositoryStatus struct {
 	// +optional
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 
-	// CannonicalName is the name of the image repository with all the
+	// CanonicalName is the name of the image repository with all the
```

Comment:
```suggestion
	// CanonicalImageName is the name of the image repository with all the
```

This was corrected wrong, it's `CanonicalImageName` per the line below the doc string